infra/nightly-resources/index.js

```diff
@@ -132,47 +132,59 @@ function groupByBenchmark(benchList) {
     return groupedBy;
 }
 
-// b may be undefined
-function compareTo(a, b) {
+// Outputs current_number - baseline_number in a human-readable format
+// If baseline_number is undefined, it will return N/A
+function getDifference(current, baseline) {
     // if b is undefined, return a
-    if (b === undefined) {
-        return tryRound(a);
+    if (baseline === undefined) {
+        return { class: "max", value: "N/A" };
     } else {
-        var difference = a - b;
+        var difference = current - baseline;
+        // if the difference is negative it will already have a "-"
         var sign = difference < 0 ? "" : "+";
+        var color = "";
+        if (difference < 0) {
+            color = "min";
+        } else if (difference > 0) {
+            color = "max";
+        }
         // put the difference in parens after a
-        return tryRound(a) + " (" + sign + tryRound(difference) + ")";
+        return { class: color, value: sign + tryRound(difference) };
     }
 }
 
-function compareAttribute(results, prevResults, attribute) {
-    const a = results[attribute];
-    const b = prevResults ? prevResults[attribute] : undefined;
-    return { class: "", value: compareTo(a, b)};
+// compare two objects at a particular attribute
+function diffAttribute(results, baseline, attribute) {
+    const current = results[attribute];
+    const baselineNum = baseline?.[attribute];
+    return getDifference(current, baselineNum);
 }
 
 const compareKeys = ["# Instructions"];
-// prevRun may be undefined
-function buildEntry(prevRun, run) {
+// baselineRun may be undefined
+function buildEntry(baselineRun, run) {
     const results = run.hyperfine.results[0];
-    const prevResults = prevRun ? prevRun.hyperfine.results[0] : undefined;
+    const baselineResults = baselineRun?.hyperfine.results[0];
 
     return {
         name: run.runMethod,
-        mean: compareAttribute(results, prevResults, "mean"),
-        min: compareAttribute(results, prevResults, "min"),
-        max: compareAttribute(results, prevResults, "max"),
-        median: compareAttribute(results, prevResults, "median"),
-        stddev: compareAttribute(results, prevResults, "stddev"),
-        
+        mean: { class: "", value: tryRound(results.mean) },
+        meanVsBaseline: diffAttribute(results, baselineResults, "mean"),
```

Comment:
nit: name `vsMain` to be more compact

Reply:
this is a dictionary right? There would be duplicate keys